lib/bundler/fetcher/compact_gem_list.rb

```diff
@@ -0,0 +1,59 @@
+module Bundler
+  class Fetcher
+    class CompactGemList
+      require 'set'
+
+      autoload :Cache,   'bundler/fetcher/compact_gem_list/cache'
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.